google/cloud/bigtable/cell.h

```diff
@@ -39,49 +85,53 @@ inline namespace BIGTABLE_CLIENT_NS {
 class Cell {
  public:
   /// Create a Cell and fill it with data.
-  Cell(std::string row_key, std::string family_name,
-       std::string column_qualifier, std::int64_t timestamp, std::string value,
-       std::vector<std::string> labels)
-      : row_key_(std::move(row_key)),
+  template <typename KeyType, typename ColumnType, typename ValueType,
+            // This function does not participate in overload resolution if
+            // ValueType is not an integral type. The case for integral types is
+            // handled by the next overload, where the value is stored as a Big
+            // Endian number.
+            typename std::enable_if<!std::is_integral<ValueType>::value,
+                                    int>::type = 0>
+  Cell(KeyType row_key, std::string family_name, ColumnType column_qualifier,
```

Comment:
I think we want `KeyType&&` here, which is will work well with the parameter being `std::forward`ed in the initialization list. Same with ValueType and ColumnType.

Reply:
Done.